javascript/ql/src/semmle/javascript/security/SensitiveActions.qll

```diff
@@ -129,11 +129,7 @@ abstract class SensitiveDataFunctionName extends SensitiveFunctionName {
 class CredentialsFunctionName extends SensitiveDataFunctionName {
   SensitiveDataClassification classification;
 
-  CredentialsFunctionName() {
-    // TODO: is it by purpose that we don't check whether `this` does not
-    // match the regexps in `notSensitive`?
-    this.regexpMatch(maybeSensitive(classification))
-  }
+  CredentialsFunctionName() { nameIndicatesSensitiveData(this, classification) }
```

Comment:
It looks OK to me. 
It effectively just adds `not this.regexpMatch(notSensitive())`.  

But I don't think I've touched this code, so someone else might have another opinion.

Reply:
I also think it is fine.
In the worst case we would lose results that would be likely FPs.

Reply:
Thanks :+1: that was my conclusion as well, I just didn't want to take the sole responsibility for this :sweat_smile: